Add desc of RedisInsight (#1187)
RedisLabs/redislabs-docs: content/ri/_index.md

alwaysopen: false
categories: ["RI"]
---
RedisInsight is a free GUI for Redis that is available on all platforms (Windows, Mac, Linux, and Docker) and works with all variants of Redis. RedisInsight allows you to:
- View performance metrics for your Redis instance with the [Overview tool]({{<relref "/ri/using-redisinsight/overview.md">}})
- View data structures visually with the [Browser]({{<relref "/ri/using-redisinsight/browser.md">}}) tool
- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management tool]({{<relref "/ri/using-redisinsight/cluster-management.md">}})
- Run commands with a REPL (read-eval-print-loop) with the the [CLI tool]({{<relref "/ri/using-redisinsight/cli.md" >}})
- Analyze memory usage with the [Memory Analysis]({{<relref "/ri/using-redisinsight/memory-analysis.md">}}) tool
- Identify and troubleshoot bottlenecks with the [Slowlog]({{<relref "/ri/using-redisinsight/slowlog.md">}}) tool
- Edit the configuration of your Redis instance with the [Configuration]({{<relref "/ri/using-redisinsight/configuration.md">}}) tool

> RedisInsight also has support for several Redis modules, including RedisGraph, RedisTimeSeries, and RediSearch.

Author: ```suggestion
RedisInsight also has support for several Redis modules, including [RedisGraph]({{<relref "/modules/redisgraph/_index.md">}}), [RedisTimeSeries]({{<relref "/modules/redistimeseries/_index.md">}}), and [RediSearch]({{<relref "/modules/redisearch/_index.md">}})
```

alwaysopen: false
categories: ["RI"]
---
RedisInsight is a free GUI for Redis that is available on all platforms (Windows, Mac, Linux, and Docker) and works with all variants of Redis. RedisInsight allows you to:

> - View performance metrics for your redis instance with the [Overview]({{<relref "/ri/using-redisinsight/overview.md">}}) tool

Reviewer: ```suggestion
- View performance metrics for your Redis instance with the [Overview tool]({{<relref "/ri/using-redisinsight/overview.md">}})
```

> - View data structures visually with the [Browser]({{<relref "/ri/using-redisinsight/browser.md">}}) tool

Reviewer: ```suggestion
- Visualize data structures with the [Browser tool]({{<relref "/ri/using-redisinsight/browser.md">}})
```

> - Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management]({{<relref "/ri/using-redisinsight/cluster-management.md">}}) tool

Reviewer: ```suggestion
- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management tool]({{<relref "/ri/using-redisinsight/cluster-management.md">}})
```

> - Use commands with a REPL (read-eval-print-loop) with the the [CLI]({{<relref "/ri/using-redisinsight/cli.md" >}}) tool

Reviewer: ```suggestion
- Run commands with a REPL (read-eval-print-loop) with the the [CLI tool]({{<relref "/ri/using-redisinsight/cli.md" >}})
```

- Analyze memory usage with the [Memory Analysis]({{<relref "/ri/using-redisinsight/memory-analysis.md">}}) tool
- Identify and troubleshoot bottlenecks with the [Slowlog]({{<relref "/ri/using-redisinsight/slowlog.md">}}) tool

> - Edit the configuration of your Redis instance with the [Configuration]({{<relref "/ri/using-redisinsight/configuration.md">}}) tool

Reviewer: Can you include "tool" in the hyperlinks?

> RedisInsight also has support for several Redis modules, including RedisGraph, RedisTimeSeries, and RediSearch.

Reviewer: Can you link to these in the docs?

> **Start using RedisInsight:**

Reviewer: ```suggestion
## Start using RedisInsight
```
